functional/test_identity_group.yml

```diff
@@ -84,4 +84,27 @@
           - functional_testing_policy_two
       register: chg_policies
 
-    - assert: { that: "{{ chg_policies.changed }} == True" }
\ No newline at end of file
+    - assert: { that: "{{ chg_policies.changed }} == True" }
+
+    - name: add external idenity group
+      hashivault_identity_group:
```

Comment:
This task won't pass until merge request #196 is merged

Reply:
next two tests depend on external group

Reply:
I know, that's why #196 needs to be merged, we can't use external group without the fix provided by it